pom.xml

```diff
@@ -187,8 +187,8 @@
          -->
         <legacy.version.org.jboss.spec.javax.security.auth.message.jboss-jaspi-api_1.1_spec>2.0.1.Final</legacy.version.org.jboss.spec.javax.security.auth.message.jboss-jaspi-api_1.1_spec>
         <legacy.version.org.jboss.spec.javax.security.jacc.jboss-jacc-api_1.5_spec>2.0.0.Final</legacy.version.org.jboss.spec.javax.security.jacc.jboss-jacc-api_1.5_spec>
-        <legacy.version.org.wildfly.security.elytron>1.20.1.Final</legacy.version.org.wildfly.security.elytron>
-        <legacy.version.org.wildfly.security.elytron-web>1.10.1.Final</legacy.version.org.wildfly.security.elytron-web>
+        <legacy.version.org.wildfly.security.elytron>1.20.2.Final</legacy.version.org.wildfly.security.elytron>
+        <legacy.version.org.wildfly.security.elytron-web>1.10.2.Final</legacy.version.org.wildfly.security.elytron-web>
```

Comment:
@fjuma just for your info, notice these two are also still used in WidlFly: https://github.com/wildfly/wildfly/blob/main/pom.xml#L584

probably not an issue at all since are only used for to compile some Wildfly modules, but, keep it in your mind.

Reply:
Thanks @yersan! To keep things consistent, I've also included those here:

https://github.com/wildfly/wildfly/pull/16064